app/helpers/automated_tests_client_helper.rb

```diff
@@ -252,13 +251,11 @@ def self.request_a_test_run(host_with_port, grouping_id, call_on, current_user,
   def self.get_test_scripts_to_run(assignment, call_on)
 
     all_scripts = TestScript.where(assignment_id: assignment.id)
-    # If the test run is requested at collection (by Admin or TA),
+    # If the test run is requested after collection (by Admin or TA),
     # All of the test scripts should be run.
-    if call_on == 'collection'
+    if call_on == 'after_collection'

```

Comment:
I see your point. It was not clear because it would be somewhat a new feature, up until now run_on_submission and run_on_request were used to identify "triggers" rather than "ownership". E.g. run_on_submission identified scripts running at every student submission, run_on_request scripts running explicitly when pressing a button (both from the student and admin side), and there was an implicit run_on_collection which would run all the scripts at collection time.